### ISSUE TITLE
Minor Issue on New Fresh Install 3

### ISSUE THREAD

**Issue by wattsie** [laravel-enso/enso#244]
This is a minor cosmetic issue...

### Prerequisites
* [YEP] Are you running the latest version?
* [YEP] Are you reporting to the correct repository?
* [YEP] Did you check the documentation?
* [YEP] Did you perform a cursory search?

### Description
When adding a new person, select "Mr" for the title, fill out other fields and hit submit.
The review screen has a blank title field.

![Screen Shot 1](https://user-images.githubusercontent.com/5244866/61503290-94925380-aa1a-11e9-9d0a-619270a71bb8.png)

### Steps to Reproduce
1. git clone
2. setup .env
3. composer install
4. yarn
5. yarn dev
6. open in browser
7. add a new person

### Expected behaviour
Add user

### Actual behaviour
Add user, but info displayed is missing the title

**Comment by aocneanu** [comment]
I cannot replicate this. Do you have any console errors?

Does the form object / title field look like this?

![image](https://user-images.githubusercontent.com/16073274/61515131-83d9e180-aa0a-11e9-820a-6c714895d151.png)

(look in the edit request response)

**Comment by wattsie** [comment]
Not sure how to get to the same as you have shown.
To me, it is like when you try to set "selected" on a standard form dropdown button, but the entry you have set does not match one in the list.

If I click on the (X) then the selections appear then I can select one.

Oh, the other interesting thing is, when looking at the person card, in the DB the title is "Mr" or #3, but the person is always showing as Female.

Very strange.

**Comment by wattsie** [comment]
Not sure if this helps.

![Screen Shot 2](https://user-images.githubusercontent.com/5244866/61524188-40f32a80-aa59-11e9-85b0-3b17d7068627.png)

**Comment by wattsie** [comment]
Just tried another computer (macOS) fresh git clone, set .env DB to SQLite data file in the database directory. 

Opened in FF and went to view admin, and the title was not selected.

Serving via "valet"

PS: group and role work as expected.

**Comment by aocneanu** [comment]
>Not sure how to get to the same as you have shown.

Go to the people table (people index) then open the browser console (ctrl shift i) and click edit on a person.

In the console go to the network tab and click the `edit` request. In the right pane select the `Preview` tab, then browse / inspect the form object

**Comment by aocneanu** [comment]
Try also creating a person on `laravel-enso.com`. The github repo's master branch is automatically deployed on `laravel-enso.com` on each commit, so it is similar to your clone.

**Comment by wattsie** [comment]
Hi @aocneanu so I have narrowed it down a little. 

Tried the docker setup, while painfully slow, did not have the issue.
(ps: npm is not installed in the docker image by default, also, db hostname has to be mysqldb)

So I moved back to local "valet" and from a sqlite db to a local mariadb, and the problem has gone away.

Therefore, there must be something strange going on with you use an sqlite db.

Cheers.
